### PR TITLE
[Cherry-pick] Update IDSDK Client to 1.1.8 from 1.1.6 (#14790)

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

Update IDSDK Client to 1.1.8 from 1.1.6 (#14790)

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -35,6 +35,7 @@
 	<PackageReference Include="Lucene.Net" Version="4.8.0-beta00016" />
     <PackageReference Include="Lucene.Net.Analysis.Common" Version="4.8.0-beta00016" />
     <PackageReference Include="Lucene.Net.QueryParser" Version="4.8.0-beta00016" />
+    <PackageReference Include="DynamoVisualProgramming.Analytics" Version="4.2.0.3421" />
```
Comment (pinzart90) [inline src/DynamoCore/DynamoCore.csproj:38]: Why are there 2 analytics pkgrefs? Looks like a bad merge

**Review by mjkkirschner — CHANGES_REQUESTED** [review]
as @pinzart90 says.

**Comment by QilongTang** [comment]
@zeusongit Is this PR still targeting 3.0 hotfix?

**Comment by zeusongit** [comment]
> @zeusongit Is this PR still targeting 3.0 hotfix? At least the 3.0.0 branch is now locked, should we re-target?

Yes, lets re-target, I will address the review comment

**Pull request closed without merging**